docs/user-docs.md

```diff
@@ -59,6 +59,10 @@ dispatcher and the history server
 
         $ dcos spark run --submit-args="--class org.apache.spark.examples.SparkPi http://downloads.mesosphere.com.s3.amazonaws.com/assets/spark/spark-examples_2.10-1.4.0-SNAPSHOT.jar 30"
 
+1.  Run a Python Spark job:
+
+        $ dcos spark run --submit-args="http://xhuynh-dev.s3.amazonaws.com/pi.py 30"
```

Comment:
It seems we shouldn't be using s3 URLS (see: https://mesosphere.atlassian.net/browse/INFINITY-575?filter=-1) -- I've been meaning to check in with you or @mgummelt about what the correct links ought to be.